corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,91 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
```

Comment:
I'm not sure about this whole `default` thing. I'm wondering if we shouldn't do it in the same way we do for things like `form.name` which is just to return *any* item if the one you are looking for isn't there:
```python
langs = request.GET['lang'] + app.langs
item.get_icon(langs)

...

def get_icon(langs):
    for lang in langs:
        if lang in media_dict and media_dict[lang]:
            return media_dict[lang]
    for lang, media in sorted(media_dict.items()):
        return media
```
See:
* https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views.py#L624-624
* https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templatetags/xforms_extras.py#L18-18

Let me know if you want to go to voice to chat about this.